lib/gui/modules/analytics.js

```diff
@@ -174,10 +98,9 @@ analytics.service('AnalyticsService', function($log, $window, $mixpanel, Setting
   this.logException = (exception) => {
     if (_.every([
       SettingsModel.get('errorReporting'),
-      isRunningInAsar(),
       errors.shouldReport(exception)
     ])) {
-      $window.trackJs.track(exception);
+      resinRaven.logException(exception);
```

Comment:
I noticed `.logException` also prints the error, so the `$log.error(exception)` would be redundant. Maybe we should only call `$log.error(exception)` on an `else` block on this conditional?